Source/WebCore/dom/ElementInlines.h

```diff
@@ -207,9 +207,9 @@ inline const AtomString& Element::getAttribute(const QualifiedName& name, const
     return getAttribute(names...);
 }
 
-inline bool isInTopLayerOrBackdrop(const RenderStyle& style, const Element* element)
+inline bool isInTopLayerOrStyleTypeIsBackdrop(const PseudoId& styleType, const Element* element)
```

Comment:
Since `PseudoId` is a scalar we should just pass it by value.

Reply:
Ok.